kctf-operator/bundle/manifests/kctf.dev_challenges.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1
```

Comment:
don't we have another dev_challenges file? what's the difference?

Reply:
its the same, it's a generated file that is moved to `/dist` by our tooling. we can `.gitignore` if helpful

Reply:
added to gitignore